package/kedro_viz/server.py

```diff
@@ -104,14 +105,17 @@ def _check_viz_up(port):
 
 
 # pylint: disable=unused-argument
-def run_viz(port=None, line=None) -> None:
+def run_viz(port=None, line=None, local_ns=None) -> None:
```

Comment:
Would users need to pass `local_ns` variable when using this line magic in the notebook?

Reply:
No, IPython will do that automatically for us thanks to https://github.com/quantumblacklabs/private-kedro/pull/639